NitroxPatcher/Patches/Dynamic/EnergyMixin_ModifyCharge_Patch.cs

```diff
@@ -16,12 +16,12 @@ public class EnergyMixin_ModifyCharge_Patch : NitroxPatch, IDynamicPatch
         public static readonly Type TARGET_CLASS = typeof(EnergyMixin);
         public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("ModifyCharge", BindingFlags.Public | BindingFlags.Instance);
 
-        public static void Postfix(EnergyMixin __instance, float amount)
+        public static void Postfix(float __result, EnergyMixin __instance)
```

Comment:
```suggestion
        public static void Postfix(EnergyMixin __instance, float __result)
```
Having these the other way round to be more inline with other patches